### DISCUSSION TITLE
[Bug] Won't install certain packages from nonportable while others from the same bucket install fine

### DISCUSSION THREAD

**Discussion by sKGu8gxV** [ScoopInstaller/Scoop#5137] [General]
I can only install spotx-np (and several other packages in nonportable) if I add a manual bucket linked to https://github.com/ScoopInstaller/Nonportable eg. "scoop add bucket somenonportable https://github.com/ScoopInstaller/Nonportable". This only seems to only affect some packages as I can install everything-np and several others without this manually added bucket from the regularly added "scoop bucket add nonportable". 

I did "scoop bucket add nonportable" and then tried "scoop install spotx-np" and get the reply "Couldn't find manifest for 'spotx-np'.". I tried removing and readding the nonportable bucket by removing"nonportable", update scoop, and readding it without change. 

My config.json:
{
    "lastupdate":  "2022-09-02T10:54:54.9606360+02:00",
    "aria2-enabled":  false,
    "SCOOP_REPO":  "https://github.com/ScoopInstaller/Scoop",
    "SCOOP_BRANCH":  "master",
    "proxy":  "my:proxy",
    "default-architecture":  "64bit",
    "rootPath":  "C:\\MyPath\\Scoop"
}

My proxy is a https proxy but I doubt the problem is there as it seems to work for everyting except some packages in nonportable. 

Proof: https://i.imgur.com/gGHLi37.png

**Comment by rashil2000** [comment]
What is the output of `scoop bucket list`?

**Reply by sKGu8gxV** [reply]
scoop bucket list

Name        Source                                             Updated             Manifests
----        ------                                             -------             ---------
extras      https://github.com/ScoopInstaller/Extras           2022-09-08 10:28:57      1676
main        https://github.com/ScoopInstaller/Main             2022-09-08 06:53:28      1073
nonportable https://github.com/TheRandomLabs/scoop-nonportable 2022-09-08 11:11:03        80

**Reply by rashil2000** [reply]
Run this:
```
scoop bucket rm nonportable
scoop bucket add nonportable https://github.com/ScoopInstaller/Nonportable
```

**Reply by sKGu8gxV** [reply]
That worked, thanks.

**Reply by sKGu8gxV** [reply]
Are you saying https://github.com/TheRandomLabs/scoop-nonportable is not official? Any easy way to find out which of my packages is installed from that bucket?

**Reply by rashil2000** [reply]
https://github.com/TheRandomLabs/scoop-nonportable has been dead for quite some time, so it was forked into https://github.com/ScoopInstaller/Nonportable and now https://github.com/ScoopInstaller/Nonportable is the official nonportable bucket.

You can list your apps and their buckets using `scoop list`

**Reply by sKGu8gxV** [reply]
Ah. I remember searching for it and not finding it easily a couple of weeks ago, guess I did a crap job. Sorry for the trouble and thanks again for the help!

**Reply by rashil2000** [reply]
No worries :)
